Catappult/appcoins-wallet-android
# Changes on sandbox flow - eskills

**What does this PR do?**

Changes layout for free to play games (sandbox)

**Database changed?**

No

**How should this be manually tested?**

Open 2048, 
Try practice mode and check that it doesn't ask for a purchase. Check buttons "OK" and "cancel".
Try to buy a ticket on "play for rewards" mode and check that in there nothing changes.


**What are the relevant tickets?**

  Tickets related to this pull-request: [ESKL-244](https://aptoide.atlassian.net/browse/ESKL-244)

**Code Review Checklist**
